apps/web/components/mvpblocks/recent-activity-feed.tsx

```
@@ -0,0 +1,180 @@
+    case "win":
+      return <Trophy className={`${iconProps} text-yellow-400`} />;
+    case "create":
+      return <Gamepad2 className={`${iconProps} text-green-400`} />;
+    case "achievement":
+      return <Trophy className={`${iconProps} text-purple-400`} />;
+    default:
+      return <Users className={`${iconProps} text-gray-400`} />;
+  }
+}
+
+function ActivityItem({ activity, index }: { activity: ActivityItem; index: number }) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ActivityItem'. Consider to delete it or rename it.

'ActivityItem' is defined here:


(lint/suspicious/noRedeclare)